bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/conf/MagLimitsId.scala

```diff
@@ -0,0 +1,10 @@
+package edu.gemini.ags.conf
+
+import edu.gemini.spModel.gemini.obscomp.SPSiteQuality.{SkyBackground, ImageQuality}
+import edu.gemini.spModel.guide.GuideSpeed
+
+sealed case class MagLimitsId(name: String)
```

Comment:
A sealed case class?  Since case classes shouldn't be extended anywhere (whether in the same file or not) maybe you meant `final`?